src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +13,101 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        if (toX != robot.getX()) {
+            if (toX > robot.getX()) {
+                lookRight(robot);
+            }
+            if (toX < robot.getX()) {
+                lookRLeft(robot);
+            }
+            while (robot.getX() != toX) {
+                robot.stepForward();
+            }
+        }
+        if (toY != robot.getY()) {
+            if (toY > robot.getY()) {
+                lookUp(robot);
+            }
+            if (toY < robot.getY()) {
+                lookDown(robot);
+            }
+            while (robot.getY() != toY) {
+                robot.stepForward();
+            }
+        }
     }
+
+    private static void lookRight(Robot robot) {
```

Comment:
do we need to add `static` to private methods?

Reply:
no) Removed